newIDE/app/src/fixtures/GDevelopServicesTestData/index.js

```diff
@@ -2463,11 +2463,6 @@ export const limitsForSilverUser: Limits = {
       max: 1000,
       limitReached: false,
     },
-    'ask-question': {
```

Comment:
Done on purpose?

Reply:
yes it's duplicated in this user's limits. I suppose this was a mistake